localStorage_value_checker_sample_app/app.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  return {
+    events: {
+      // Framework Events
+      'app.created':'checkLocalStorage', // Call 'checkLocalStorage' function below
+      // Request Events (None)
+      // DOM events
+      'keyup #inputValueId': function(event){ // User clicked the 'enter'/'return' button so store value in localStorage
+        if(event.keyCode === 13)
+          return this.processInputValue();
+      },
+      'click .reset': 'resetValue' // Call 'resetValue' function below
+    },
+    checkLocalStorage: function () { // Look in localStorage for if value is present for 'exampleString'
+        if (this.store('exampleString') === null) { // If no value
```

Comment:
For this being an educational app aimed at beginners/etc is why I'd included the comment "If not value" but for sure I'll get rid of it np